### ISSUE TITLE
Add search engine not work if is default engine and hidden

### ISSUE THREAD

**Issue by pulcov** [mozilla/policy-templates#263]
Hi,

Add engine google or other default engine not work if this engine has been deleted by user. You can see more information on bugzilla [#333069](https://bugzilla.mozilla.org/show_bug.cgi?id=363069)

**Comment by pulcov** [comment]
No. Engine cannot be duplicated. But, when user remove a default search engine, it is not deleted from disk but is set has hidden.

If you want to add the same engine again :
- The expected behavior : engine is shown again.
- Actual behavior : nothing is happening.

**Comment by mkaply** [comment]
What are you setting for the engine values in the policy? There's not really a way to put "Google" in teh search engine policy.

**Comment by mkaply** [comment]
See above comment @pulcov
